### PR TITLE
Dynamic email object in `_topbar.html` partial

### PR DESCRIPTION
**Pull request by enoren5**
This is my effort to try to wire up a dynamic Charcter Field text box to house an email address in the  `_topbar.html` partial. I have defined a brand new `Email` Class and migrated the models. I enabled the Email app in the Admin Dashboard and attempted to populate the Jinja template with conditional control logic to call the object. I logged into the Admin Dashbaord and added a placeholder email address.

With all of the above changes, for some reason the partial is not serving the email address. I've tried many different combinations by pluralizing the first instance of `email` as `emails` while double checking that this matches what is inside `accounts/views.py` but no matter what I try, I can't get the email to show up.

@abubakarA-Dot: Could you take a look at these diffs and experiment with what I have started here and try to get it working?

Thanks.

### CODE REVIEW

**Review by abubakarA-Dot — CHANGES_REQUESTED** [review]
I have just reviewed your code and you have done it amazingly but at the end in accounts/views.py file, on line 115, you have to pass a parameter to get a certain user's email when he/she is logged in, so what you to do is, 
`def topbat_email(request, id):
    try:
        email = get_object_or_404(Email, pk=id)
    except Email.DoesNotExist:
        email = None
    context = {
        'email':email
      }
    return render(request, 'partials/_topbar.html', context)`
and then in templates/partials/_topbar.html file:
`
{% if email %}
{{email.email}}
{% else %}
<p>No Email</p>
{% endif %}
`
I think that's what you want to get as a result. if that's what you want to get let me know please.

**Comment by abubakarA-Dot** [comment]
and most important thing, you also need to pass id into your URL. example
`path('email', viewstopbar_email, name='email').`

**Comment by enoren5** [comment]
I'm not trying to be rude or mean but it seems like you don't have much experience with GitHub because I find your code snippet to be extremely hard to follow. The changes you've suggested for the `topbar_email()` function you've written all on a single line even though it spans across 5 lines. I can't understand your code. Would it be possible for you to rewrite your code sample using proper blockquote multi-line styling? Also, could you make the changes and push to this branch which is being tracked remotely so that I may download it and test it out locally on my machine?

**Review by abubakarA-Dot — CHANGES_REQUESTED** [review]
for templates/partials/_topbar.html

replace:
{% if emails %}
    {{emails.email}}
{% endif %}
with
{% if emails %}
  {% for email in emails %}
      {{ email.email }}
  {% endfor %}
{% endif %}

**Comment by enoren5** [comment]
Your code snippet is still not formatted correctly but I see what you are trying to convey. I copied and paste your code into my `_topbar.html` and the email still is not parsing. 

More changes are needed. 

You wrote recommend that I could try this: `def topbat_email(request, id): try: email = get_object_or_404(Email, pk=id) except Email.DoesNotExist: email = None context = { 'email':email } return render(request, 'partials/_topbar.html', context)` which is (completely) unhelpful because you need to format it so it is easier to read across 5 lines rather than across a single line. Please make your code easier for me to read by using proper blockquote formatting. 

Also, could you write all of your proposed changes and push them to this particular branch in separate commits?

**Comment by enoren5** [comment]
Resolved in #179
